logging/cluster-logging-upgrading.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} 4.7 and 4.8 support OpenShift Logging 5.0, 5.1, and 5.2.
+{product-title} 4.7, 4.8, and 4.9 support OpenShift Logging 5.0, 5.1, and 5.2.
```

Comment:
@rolfedh, PTAL at my guess at the logging updates?

Reply:
/lgtm

Reply:
@anpingli Would you review and approve this change for OpenShift 4.9 GA?